Fix/improve for mpc

1-Bart-1 opened the pull request:
Fix bugs and add changes that improve the accuracy of the 3 line model and compatibility for https://github.com/Albatross-Kite-Transport/KitePredictiveControl.jl

ufechner7 commented:
Do I understand correctly that you add a new, simplified model for control purposes? If that is the case, can you perhaps change the title of the pull request and add some tests?

1-Bart-1 commented:
No, I just fixed a bug on the 3 line model.

1-Bart-1 commented:
No, the model still works the same. I found some mistake in the flap angle formula that I fixed, and updated some other things. I just had to move the code around a bit to be able to work with it from the other package, but the model is still the same otherwise.

1-Bart-1 commented:
The simplified model just exists inside KitePredictiveControl.jl

ufechner7 commented:
So do you think this pull request is ready to be merged?
